Python/Product/Django/Django.csproj

```diff
@@ -264,12 +264,17 @@
   <Choose>
     <When Condition="$(IncludeDjangoHtmlExtensions) == 'true'">
       <ItemGroup>
-        <Reference Include="Microsoft.Web.Core" />
-        <Reference Include="Microsoft.Web.Editor" />
-        <Reference Include="Microsoft.Html.Core" />
-        <Reference Include="Microsoft.Html.Editor" />
-        <Reference Include="Microsoft.VisualStudio.Html.Package" />
         <Reference Include="Microsoft.VisualStudio.Web.HTML" />
+        <Reference Include="Microsoft.Web.Core" Condition=" '$(VSTarget)' == '15.0' " />
+        <Reference Include="Microsoft.Web.Editor" Condition=" '$(VSTarget)' == '15.0' " />
+        <Reference Include="Microsoft.Html.Core" Condition=" '$(VSTarget)' == '15.0' " />
+        <Reference Include="Microsoft.Html.Editor" Condition=" '$(VSTarget)' == '15.0' " />
+        <Reference Include="Microsoft.VisualStudio.Html.Package" Condition=" '$(VSTarget)' == '15.0' " />
+        <Reference Include="Microsoft.WebTools.Languages.dll" Condition=" '$(VSTarget)' == '16.0' " />
```

Comment:
Out of curiosity, how did you discover the references required for 15.0 as opposed to 16.0?

Reply:
There was a conversion table included in the email, I'll forward it to you.